### ISSUE TITLE
Support for flyway-spring-test '6.0.0' 

### ISSUE THREAD

**Issue by vinits91** [zonkyio/embedded-database-spring-test#65]
We just updated our flyway-spring-test to '6.0.0' version and integration tests started failing with embedded-database-spring-test '1.5.1'  and Spring boot version is 2.1.8 with following 

`10:30:57 Caused by:
10:30:57 java.lang.NoClassDefFoundError: org/flywaydb/core/api/configuration/FluentConfiguration`

embedded-database-spring-test is compatible with latest flyway-spring-test version ?, if not when can we expect update on embedded-database-spring-test ?

**Comment by tomix26** [comment]
Hi, are you sure you are using a correct version of flyway-core dependency? Because the `org.flywaydb.core.api.configuration.FluentConfiguration` class has been introduced in `flyway-core:5.1.0` and should be still present. Also note that Flyway 6 is only supported by Spring Boot 2.2, here is more details: https://github.com/spring-projects/spring-boot/issues/18483.

When I run integration tests of `embedded-database-spring-test` component, all tests passed to versions `flyway-core:6.0.2` and `flyway-spring-test:6.0.0`. Since version `6.0.3`, there is some incompatibility that will need to be resolved but not related to your original question.
```
java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at io.zonky.test.db.flyway.OptimizedFlywayTestExecutionListener.createMigrationResolver(OptimizedFlywayTestExecutionListener.java:268)
	at io.zonky.test.db.flyway.OptimizedFlywayTestExecutionListener.resolveMigrations(OptimizedFlywayTestExecutionListener.java:239)
	at io.zonky.test.db.flyway.OptimizedFlywayTestExecutionListener.findFirstVersion(OptimizedFlywayTestExecutionListener.java:221)
	at io.zonky.test.db.flyway.OptimizedFlywayTestExecutionListener.isAppendable(OptimizedFlywayTestExecutionListener.java:211)
	at io.zonky.test.db.flyway.OptimizedFlywayTestExecutionListener.reloadDataSource(OptimizedFlywayTestExecutionListener.java:182)
	at io.zonky.test.db.flyway.OptimizedFlywayTestExecutionListener.optimizedDbReset(OptimizedFlywayTestExecutionListener.java:148)
	at io.zonky.test.db.flyway.OptimizedFlywayTestExecutionListener.beforeTestMethod(OptimizedFlywayTestExecutionListener.java:114)
```

**Comment by tomix26** [comment]
I've created a separate issue (#67) to fix the compatibility with Flyway 6.0.3+.

**Comment by tomix26** [comment]
No response, I'm closing the issue.
